src/components/FormFieldWrapper/index.js

```diff
@@ -14,6 +15,33 @@ const getClassname = (element, error) => {
   ].join(' ');
 };
 
+const getChildren = (config, helper) => {
+  switch (config.element) {
+    case 'Autocomplete':
+      return (
+        <div className={styles['autocomplete-wrapper']}>
+          <Autocomplete
+            helper={helper}
+            {...config}
+          />
+        </div>
+      );
+
+    default:
+      return (
+        <config.element
+          className={getClassname(config.element, fieldIsInvalid(helper))}
+          id={config.name}
+          // pass props from config e.g. type, placeholder, maxlength
+          {...config.html}
+          // domOnlyProps required with latest react and redux-form 5.x
+          // see: https://github.com/erikras/redux-form/issues/1441#issuecomment-236966387
+          {...domOnlyProps(helper)}
+        />
+      );
+  }
+};
```

Comment:
@mattberridge 
I don‘t think I could‘ve done this differently?
If we ever add input types (like the image picker), we‘ll simply add the necessary wrappers and actual element in the switch.